components/BottomAppBar/examples/BottomAppBarNavBarExample.m

```diff
@@ -0,0 +1,158 @@
+  UINavigationItem *navItem = [[UINavigationItem alloc] initWithTitle:@""];
+  [self.view addSubview:self.navBar];
+
+  // Configure the navigation buttons to be shown on the bottom app bar.
+  UIBarButtonItem *barButtonLeadingItem =
+      [[UIBarButtonItem alloc] initWithTitle:@"Back"
+                                       style:UIBarButtonItemStylePlain
+                                      target:self
+                                      action:@selector(didTapLeft:)];
+
+  UIBarButtonItem *barButtonTrailingItem =
+      [[UIBarButtonItem alloc] initWithTitle:@"Right"
+                                       style:UIBarButtonItemStylePlain
+                                      target:self
+                                      action:@selector(didTapRight:)];
+
+  navItem.leftBarButtonItems = @[ barButtonLeadingItem ];
+  navItem.rightBarButtonItems = @[ barButtonTrailingItem ];
+
+  [self.navBar setLeftBarButtonItems:navItem.leftBarButtonItems];
+  [self.navBar setRightBarButtonItems:navItem.rightBarButtonItems];
+
+  MDCBasicColorScheme *clearScheme =
+      [[MDCBasicColorScheme alloc] initWithPrimaryColor:[UIColor clearColor]];
+  [MDCNavigationBarColorThemer applyColorScheme:clearScheme toNavigationBar:self.navBar];
+  self.navBar.tintColor = [UIColor blackColor];
+}
+
+- (void)viewWillLayoutSubviews {
+  [super viewWillLayoutSubviews];
+  self.navBar.frame = self.view.bounds;
+}
+
+- (void)didTapLeft:(id)sender {
+  [self.navigationController popViewControllerAnimated:YES];
+}
+
+- (void)didTapRight:(id)sender {
+  NSLog(@"Right button tapped.");
+}
+
+@end
+
+@interface BottomBarNavBarExample : MDCBottomAppBarViewController
+
+@end
+
+@implementation BottomBarNavBarExample
+
+- (id)init {
+  self = [super init];
+  if (self) {
+    self.title = @"Bottom App Bar Nav Bar";
+    [self commonBottomBarNavBarExampleInit];
+  }
+  return self;
+}
+
+- (void)commonBottomBarNavBarExampleInit {
+  self.viewController = [[BottomAppBarExampleTableViewController alloc] init];
+  self.contentViewController = [[BottomAppBarNavBarViewController alloc] init];
+  self.floatingButtonPosition = MDCBottomAppBarFloatingButtonPositionCenter;
+
+  // Add touch handler to the floating button.
+  [self.floatingButton addTarget:self
+                          action:@selector(didTapFloatingButton:)
+                forControlEvents:UIControlEventTouchUpInside];
+
+  // Set the image on the floating button.
+  [self.floatingButton setImage:[self floatingButtonImage] forState:UIControlStateNormal];
+
+  // Theme the floating button.
+  MDCBasicColorScheme *whiteScheme =
+      [[MDCBasicColorScheme alloc] initWithPrimaryColor:[UIColor whiteColor]];
+  [MDCButtonColorThemer applyColorScheme:whiteScheme toButton:self.floatingButton];
+}
+
+- (void)didTapFloatingButton:(id)sender {
+  [self setFloatingButtonVisible:NO animated:YES];
+  dispatch_after(dispatch_time(DISPATCH_TIME_NOW, 2 * NSEC_PER_SEC), dispatch_get_main_queue(), ^{
```

Comment:
This is a good place for a comment on why it's leaving and when it should come back IRL.

Reply:
Adding comment.